Formula/molecule.rb

```diff
@@ -104,23 +109,18 @@ class Molecule < Formula
   end
 
   resource "cryptography" do
-    url "https://files.pythonhosted.org/packages/9b/77/461087a514d2e8ece1c975d8216bc03f7048e6090c5166bc34115afdaa53/cryptography-3.4.7.tar.gz"
-    sha256 "3d10de8116d25649631977cb37da6cbdd2d6fa0e0281d014a5b7d337255ca713"
+    url "https://files.pythonhosted.org/packages/cc/98/8a258ab4787e6f835d350639792527d2eb7946ff9fc0caca9c3f4cf5dcfe/cryptography-3.4.8.tar.gz"
+    sha256 "94cc5ed4ceaefcbe5bf38c8fba6a21fc1d365bb8fb826ea1688e3370b2e24a1c"
   end
 
   resource "distro" do
     url "https://files.pythonhosted.org/packages/a5/26/256fa167fe1bf8b97130b4609464be20331af8a3af190fb636a8a7efd7a2/distro-1.6.0.tar.gz"
     sha256 "83f5e5a09f9c5f68f60173de572930effbcc0287bb84fdc4426cb4168c088424"
   end
 
-  resource "docker-py" do
-    url "https://files.pythonhosted.org/packages/fa/2d/906afc44a833901fc6fed1a89c228e5c88fbfc6bd2f3d2f0497fdfb9c525/docker-py-1.10.6.tar.gz"
-    sha256 "4c2a75875764d38d67f87bc7d03f7443a3895704efc57962bdf6500b8d4bc415"
-  end
```

Comment:
If we're removing this we need to also remove it from `pypi_formula_mappings.json`.

Reply:
I wasn't aware `pypi_formula_mappings.json` needed to be updated, but I will do that.  When adding new resources do they need to be added to the mappings as well?  Finally, is this something that should be added to the [Python for Formula Authors](https://docs.brew.sh/Python-for-Formula-Authors) docs or is it documented somewhere else and just needs to be linked?

Reply:
It depends. If they're already found as default dependencies by `pipgrip`, then it's not needed. If these are additional Python modules that aren't found via `pipgrip`, they need to be added there.